Tests/RevenueCatUITests/CustomerCenter/ManageSubscriptionsViewModelTests.swift

```diff
@@ -171,19 +390,76 @@ final class MockManageSubscriptionsPurchases: ManageSubscriptionsPurchaseType {
 }
 
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
-private extension ManageSubscriptionsViewModelTests {
+private class Fixtures {
+    private init() {}
```

Comment:
Same here about the empty lines at the beginning and the bottom in this class and subclasses

by the way, this Fixtures class is neat!

Reply:
Thanks 😄 

Should be fixed: https://github.com/RevenueCat/purchases-ios/pull/4046/commits/89d4a8d61b50b0a04c8ff82c97b3d8d7b9d5526d